squash/dashboard/models.py

```diff
@@ -1,18 +1,22 @@
 import json
 from django.db import models
 from django.contrib.auth.models import User
-from .bokeh_utils import update_bokeh_sessions
 
 
 class Job(models.Model):
     """Job information"""
     STATUS_OK = 0
     STATUS_FAILED = 1
 
-    ci_name = models.CharField(max_length=32, blank=False,
-                               help_text='Name of the Jenkins job')
     ci_id = models.CharField(max_length=16, blank=False,
                              help_text='Jenkins job ID')
+    ci_name = models.CharField(max_length=32, blank=False,
+                               help_text='Name of the Jenkins project,\
```

Comment:
An elegant way of dealing with multi-line strings that are wrapped in parentheses:

```
CharField(help_text='Name of the Jenkins project, '
                                   'e.g. validate_drp')
```

Python auto-concatenates those two strings.